package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fulcrumapp/fulcrum-schema",
-  "version": "3.6.2",
+  "version": "3.7.2",
```

Comment:
Why 3.7.2? It should just be 3.7.0 since it is a new non-breaking feature.

Reply:
yes you are right